### ISSUE TITLE
No Changelog for 23.0.0

### ISSUE THREAD

**Issue by Delphinator** [nextcloud/server#30097]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Go to the [server download page](https://nextcloud.com/install/#instructions-server). "Latest stable version" is 23.0.0.
2. Follow the link to the [changelog](https://nextcloud.com/changelog/). Latest version with a changelog entry is 22.2.3.
3. [Latest release](https://github.com/nextcloud/server/releases/latest) here on GitHub is 23.0.0 (with no description) from eight days ago.

This happened with at least v20.0.0 (see #23219) and v20.0.8 (see #25792) as well.

### Expected behaviour
The changelog should be updated before or very shortly after a release is made - especially a new major version. Having no published changelog for the current stable version for over a week is very odd for a project as big as nextcloud. Please consider, that a new major version indicates possible breaking changes admins might want to know about before upgrading and not finding one will likely delay updates and reduce confidence in the project.

### Actual behaviour
I was unable to find a changelog for 22.2.3 to 23.0.0

**Comment by acsfer** [comment]
Cc @jospoortvliet

**Comment by morremeyer** [comment]
I agree with what @Delphinator points out and collected some data points to illustrate the issue

# Timeline

* The [`v23.0.0`](https://github.com/nextcloud/server/tree/v23.0.0) tag was set 2021-11-26, 20:49Z
* The [blog post](https://nextcloud.com/blog/nextcloud-hub-2-brings-major-overhaul-introducing-nextcloud-office-p2p-backup-and-more/) came out 2021-11-30
* The [`v23.0.0` Release](https://github.com/nextcloud/server/releases/tag/v23.0.0) was published 2021-12-03, 21:58Z
* The `nextcloud:23.0.0-apache` Docker image was published to hub.docker.com at 2021-12-02 07:05Z the latest (my automated upgrading tool picked it up at that time).
* As of writing (2021-12-07 09:00Z), there is no changelog released on https://nextcloud.com/changelog/ nor in the repository (the repository CHANGELOG.md just links to https://nextcloud.com/changelog/)

**Comment by rawtaz** [comment]
Missing it too. Kind of hard to review changes before upgrading when there's no changelog :D

**Comment by NicolasGoeddel** [comment]
My own Nextcloud installation reminds me everyday to upgrade to version 23 but without a changelog I don't want to. And now it is quite a while.
Sorry for the less relevant information. I just wanted to point out that there should be a changelog if the automatic update checker suggests to update.

**Comment by ohthehugemanatee** [comment]
Big +1 here. Nice announcement and all, but the changelog is essential for a lot of us.

**Comment by rawtaz** [comment]
This is quite funny by now. How can it be this hard for a company like Nextcloud to produce a few lines of changelog, the same way they have done for every release a long time back in time? Seriously xD

**Comment by morremeyer** [comment]
Please stop posting comments that do not add new information.

If you agree, +1 the issue.

This is a technical issue, not a discussion forum for opinions. For discussion about opinions, please go to hackernews, reddit or somewhere else.

Each unnecessary comment here triggers unnecessary notifications for 20+ people now, and that number is only going up. 

To everyone getting a useless notification from this: Sorry for that. Apparently, a reminder in the issue itself is not enough.

**Comment by rawtaz** [comment]
Perhaps that's what you get for not doing one of the absolutely most simplest possible things in a software project, that you have evidently been able to do before. It's been over a week now. How hard can it be. What is stopping you?

This will be my last comment, I totally understand that it's not helping. It's just silly that it's a special case, this one. Thanks.

**Comment by morremeyer** [comment]
There is now a changelog entry for 23.0.0 on nextcloud.com/changelog, but it just links to the blog post released 2021-11-30.

A full list of changes is still not available.
